apps/extension/package.json

```diff
@@ -35,7 +35,7 @@
     "@penumbra-zone/types": "workspace:*",
     "@penumbra-zone/ui": "workspace:*",
     "@penumbra-zone/wasm": "workspace:*",
-    "@tanstack/react-query": "^5.28.9",
+    "@tanstack/react-query": "4.36.1",
```

Comment:
This is somewhat sad. `osmos-query` is currently pinned on an old version. This means we need to downgrade until the team has updated theirs. We've flagged this with them and they are prioritizing it.